.changeset/nice-crabs-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': major
+---
+
+Removed `iui-tooltip-container` and `iui-tooltip-visible` classes. The display is now toggled using the `hidden` HTML attribute, and the positioning should be managed using custom JS/CSS.
```

Comment:
Should there be a mention about _"positioning"_? Because I believe even before this PR, positioning had to be handled separately.

Also, might be useful to mention that the tooltip is now always in the DOM.

Reply:
> Should there be a mention about _"positioning"_? Because I believe even before this PR, positioning had to be handled separately.

it was partly managed with `iui-tooltip-container`. theoretically, you could easily calculate the inset values as they would be relative to the container, whereas now we don't provide a container

> Also, might be useful to mention that the tooltip is now always in the DOM.

this part is kinda implied by `hidden`, but also it's not a requirement. it can still be removed from the DOM if desired (this is the css changeset)